src/services/kubelet-service.js

```diff
@@ -134,6 +141,14 @@ const kubeletUpdatePod = async function (uploadPodData, fogNodeUuid, user, trans
       ports: ms.ports || [],
       routes: ms.routes || [],
     }
+    if (ms.env && ms.env.length > 0) {
```

Comment:
duplicate checks? both needed? if yes, would it be helpful if we make it a function?

Reply:
updatePod is not done yet. Most of its logic is similar to createPod, I'll optimize it later.